Meter.c

```diff
@@ -340,11 +340,11 @@ static void LEDMeterMode_draw(Meter* this, int x, int y, int w) {
       mvaddnstr(yText, x, caption, w);
    }
 
-   int captionLen = strlen(caption);
-   if (w <= captionLen) {
+   int captionWidth = (int)strnlen(caption, (unsigned int)w);
```

Comment:
```suggestion
   int captionWidth = (int)strnlen(caption, (size_t)w);
```

(Note the remark re an additional `assert` above)